testing-common/src/main/groovy/io/opentelemetry/instrumentation/test/base/HttpClientTest.groovy

```diff
@@ -341,34 +341,14 @@ abstract class HttpClientTest<REQUEST> extends InstrumentationSpecification {
 
   //FIXME: add tests for POST with large/chunked data
 
-  def "trace request without propagation"() {
```

Comment:
I think this only tested that `TestHttpServer` respects is-test-server, nothing about the client itself.